components/sysutils/multitail/manifests/sample-manifest.p5m

```diff
@@ -0,0 +1,36 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2023 <contributor>
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.human-version value=$(HUMAN_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file path=usr/bin/multitail
+file path=usr/etc/multitail.conf.new
+file path=usr/etc/multitail/conversion-scripts/colors-example.pl
```

Comment:
usr/etc is wrong.
Furthermore, you also want preserve=true (or similar) for configuration files.

Reply:
But this is the sample (generated by gmake). The effective manifest is multitail.p5m.

Reply:
Ok, but if the manifest uses a wrong folder for the configuration it's quite likely that the real application is also looking at the wrong path for its configuration. You should use settings in the Makefile to fix this.